jerry-debugger/src/lib/protocol-handler.ts

```diff
@@ -396,6 +467,70 @@ export class JerryDebugProtocolHandler {
     return this.lastBreakpointHit;
   }
 
+  getScriptIdByName(name: string) {
+    // skip the first (dummy) source
```

Comment:
What's the dummy for?

Reply:
Perhaps a dumb way to do it, I was of two minds on it - I put an unused dummy object into the array to allow 1-indexed values to be used as is. I.e. the line numbers coming from JrS start with 1, and the script IDs I'm assigning start with 1.

Reply:
Ah ok, fair enough.
Maybe mention it in a comment with the instance member declaration?